addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -854,12 +854,12 @@ export const editorCommands = {
                 // Add extra columns.
                 const restore = preserveCursor(editor.document);
                 for (const column of columns) {
-                    column.className = column.className.replace(REGEX_BOOTSTRAP_COLUMN, `col$1-${columnSize}`);
+                    column.className = column.className.replace(REGEX_BOOTSTRAP_COLUMN, `col$1-${columnSize} `);
```

Comment:
Why is the space needed here? Does the regex include the trailing space?

Reply:
without the space `col-6 ps-0` will become `col-6ps-0`